### DISCUSSION TITLE
GSSOC '21 Project

### DISCUSSION THREAD

**Discussion by APratham** [vigneshshettyin/Flask-Generate-Certificate#23] [General]
This project is a part of [Girlscript Summer of Code 2021](https://gssoc.girlscript.tech/). So which means that by default maintainers and admins will add the GSSOC label to all issues and PRs. If you want to contribute outside of the program, you can do so by mentioning on the issue or else commenting here so we know. If you are a part of GSSOC you have to follow these rules for contributing: 

(These are general rules for contributing anywhere, I am just formally putting them down here)

1. Once you create an issue, you have to wait for maintainers/admins to assign it to you _before_ starting work on it. If you don't wait and immediately create a PR, it can get **rejected** also. 
2. All issues have to contain the GSSOC label. If your issue doesn't contain it, you can _ask_ to have it included. 
3. We have different levels based on the _complexity_ of an issue. Remember to only contribute to issues which you think are of your liking, do not contribute and finish an issue with a lower level _just because you can_. 
4. The maintainers and admins will also create some issues for everyone else to contribute. If you are interested, you can just comment on the issue and have it assigned to you _before_ starting work on it.

**Comment by vigneshshettyin** [comment]
Kindly note simple issues has to be closed within 2 days. Unresolved issues will be assigned to others.

Happy Coding!
